pb/p2pd.proto

```diff
@@ -2,6 +2,7 @@ package p2pd.pb;
 
 message Request {
   enum Type {
+    IDENTIFY       = 0;
```

Comment:
do we want this in the 0 position? other than this, looks good.

Reply:
It's kind of the most basic operation, that's why I put it there. Plus the enums where not proto3 safe, they needed the first element to be 0 so I commandeered it.